docs/reference/esql/source-commands/from.asciidoc

```diff
@@ -82,3 +82,11 @@ Use the optional `METADATA` directive to enable <<esql-metadata-fields,metadata
 ----
 FROM employees METADATA _id
 ----
+
+Use single quotes (`"`) or tripple quotes (`"""`) to escape the index name, if
+it contains special characters:
```

Comment:
```suggestion
Use enclosing double quotes (`"`) or three enclosing double quotes (`"""`) to escape index names
that contain special characters:
```

Reply:
Rationale behind this suggestion: It's a bit confusing to say *single quote*, which usually means `'` (apostrophe).

The "enclosing double quotes" phrasing is from an [EQL doc page](https://www.elastic.co/guide/en/elasticsearch/reference/current/eql-syntax.html#eql-syntax-escape-an-event-category) and just seems a little clearer.